en/release-notes/asset-manager-upgrade-guide.md

```diff
@@ -1,344 +1,350 @@
 
-In order to provide a smooth upgrade experience, we have retained compatibility with the `cc.loader` related APIs, and most of the game project can run as usual, except for some special uses that are not compatible and must be manually upgraded. We will then remove full compatibility with `cc.loader` when the time comes. If you are temporarily uncomfortable upgrading due to the project cycle, etc., you can keep the original writing while making sure the test passes.
+Therefore, Creator in **v2.4** introduced a new resource management module -- **Asset Manager**. Compared to the previous `cc.loader` module, **Asset Manager** not only provides better loading performance, but also support **Asset Bundle**, preload resources and more convenient resource release management. And **Asset Manager** also has strong extensibility, which greatly improves the development efficiency and user experience of developers. We recommend that all developers upgrade.
 
-Currently, when using those old APIs, the engine outputs a warning and suggests an upgrade method. Please adjust the code according to the warnings and upgrade to the new usage. Unfortunately, due to an upgrade of the underlying layer, we have left behind a few incompatible APIs that will output error messages while running. If you've already decided on an upgrade, then the following is what you need to read carefully.
+To bring a smooth upgrade experience, we will maintain compatibility with `cc.loader` related APIs, and most of the game project can run as usual, except for a few that use incompatible special usage APIs that must be manually upgraded. And we will then remove full compatibility with `cc.loader` when the time comes. If you are temporarily uncomfortable upgrading due to the project cycle, etc., you can keep the original writing while making sure the test passes.
 
-For **Artist and Game Designer**, all resources in the project, such as scenes, animations, Prefab, do not need to be modified or upgraded. <br>
-For **Programmers**, the impact is mainly in the fact that all APIs that were originally used in the code for `cc.loader` need to be changed to a series of APIs that use `cc.assetManager`, which is described in detail in this article.
-**Note**: Since v2.4 supports Asset Bundle feature, the subpackage settings in the project need to be upgraded, please refer to [Subpackage Upgrade Guide](./subpackage-upgrade-guide.md) documentation.
+Currently, when using those old APIs, the engine outputs a warning and suggests an upgrade method. Please adjust the code according to the warnings and the instructions in this document and upgrade to the new usage. Unfortunately, due to an upgrade of the underlying layer, we have left behind a few incompatible APIs that will output error messages while running. If you have decided to make the upgrade, then please read the following carefully.
```

Comment:
the engine outputs a warning and suggests an upgrade method. -> the engine will output warnings and suggestions for upgradation.